djangocms_navigation/views.py

```diff
@@ -98,3 +99,12 @@ def get_data(self):
                     queryset = queryset.filter(**options)
 
         return queryset
+
+
+class MessageStorageView(View):
```

Comment:
Is this replaced by the fact that we are happy to let the page reload or is it still required?

Reply:
Because of the changed default expand/collapse, I've had to avoid a page reload. So this endpoint is needed to query the django messages queue that Treebeard updates when it moves nodes.